packages/url_launcher/android/src/main/java/io/flutter/plugins/urllauncher/WebViewActivity.java

```diff
@@ -0,0 +1,124 @@
+package io.flutter.plugins.urllauncher;
+
+import android.app.Activity;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.os.Build;
+import android.os.Bundle;
+import android.provider.Browser;
+import android.view.KeyEvent;
+import android.webkit.WebResourceRequest;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+import java.util.HashMap;
+import java.util.Map;
+
+/*  Launches WebView activity */
+public class WebViewActivity extends Activity {
+
+  public static String ACTION_CLOSE = "close action";
```

Comment:
nit: Some Javadoc explaining what this constant is used for would be useful. https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required

I didn't realize spaces were supported here. Neat.